### DISCUSSION TITLE
file picker behavior

### DISCUSSION THREAD

**Discussion by gdkrmr** [helix-editor/helix#8980] [General]
I have found a usability issues with the file picker:

- if `hidden = true`, then editing the config is very tedious because you simply cannot open hidden directories
- if `hidden = false` file picker goes through way too many directories.

I suggest to 
- limit the scope of file picker to the typed directory, e.g. typing `.config/` should stop file picker from searching paths outside of `.config/` and
- it should also be possible to search hidden directories, even if `hidden = true` when the directory name is explicitly typed, e.g. `.config/` should search this path even with `hidden = true`.

**Comment by David-Else** [comment]
I am giving this a go at the moment, might be of interest:

```
[keys.normal.space.t]
"." = ":toggle file-picker.hidden"
```

**Comment by kirawi** [comment]
Try `^.config/`
https://github.com/junegunn/fzf#search-syntax
